src/main/EmailService.java

```diff
@@ -0,0 +1,42 @@
+package main;
+
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import static lombok.AccessLevel.PRIVATE;
+
+
+@Service
+@RequiredArgsConstructor
+@FieldDefaults(level = PRIVATE)
+public class EmailService {
+    @Autowired
+    EmailsDataBase emailsDataBase;
+
+    public EmailService(EmailsDataBase emailsDataBase) {
```

Comment:
Better declare EmailsDataBase final and use RequriedArgsConstructor, no need to include this boilerplate code here